pogom/search.py

```diff
@@ -61,47 +52,61 @@ def send_map_request(api, position):
         log.warning("Uncaught exception when downloading map " + str(e))
         return False
 
+def get_new_coords(init_loc, distance, bearing):
+    """ Given an initial lat/lng, a distance(in kms), and a bearing (degrees),
+    this will calculate the resulting lat/lng coordinates.
+    """ 
+    R = 6378.1 #km radius of the earth
```

Comment:
__[flake8]__
* [E261] at least two spaces before inline comment
* [E262] inline comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->